client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -52,11 +53,12 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_2, value ) }
 					/>
 
-					<FormCountrySelect
+					<ComboboxControl
 						name="country_code"
```

Comment:
The `name` prop does not exist on `ComboboxControl`.

Reply:
d0ffd5a